src/groups.c

```diff
@@ -88,7 +88,7 @@ int main (int argc, char **argv)
 	GETGROUPS_T *groups;
 
 	sys_ngroups = sysconf (_SC_NGROUPS_MAX);
-	groups = (GETGROUPS_T *) malloc (sizeof (GETGROUPS_T) * sys_ngroups);
+	groups = (GETGROUPS_T *) xmalloc (sizeof (GETGROUPS_T) * sys_ngroups);
```

Comment:
You forgot to paste the compiler error for this one.  Anyway, it seems correct.

Reply:
There was none; came up by manual review.